smart-contracts/contracts/mixins/MixinLockManagerRole.sol

```diff
@@ -0,0 +1,47 @@
+pragma solidity ^0.5.0;
+
+import '@openzeppelin/upgrades/contracts/Initializable.sol';
```

Comment:
Could you add a comment RE the source of this?  It's clearly copy-paste and then edit with our naming.  We should include a shoutout

Reply:
YUp. On it.